### ISSUE TITLE
revise bounds on Hackage to allow latest deepseq

### ISSUE THREAD

**Issue by vmchale** [visq/language-c#95]
Right now `language-c` can't be built with GHC 9.8:

```
Error: cabal: Could not resolve dependencies:
[__0] trying: language-c-0.9.2 (user goal)
[__1] next goal: process (dependency of language-c)
[__1] rejecting: process-1.6.18.0/installed-b188 (conflict: language-c =>
deepseq>=1.4.0.0 && <1.5, process => deepseq==1.5.0.0/installed-5710)
[__1] trying: process-1.6.18.0
[__2] trying: unix-2.8.3.0/installed-7905 (dependency of process)
[__3] next goal: time (dependency of unix)
[__3] rejecting: time-1.12.2/installed-c858 (conflict: language-c =>
deepseq>=1.4.0.0 && <1.5, time => deepseq==1.5.0.0/installed-5710)
```

**Comment by frasertweedale** [comment]
*language-c* is a dependency of *c2hs*, which is a build dependency of... quite a few packages, I think.  It would be great to resolve this soon.

Thanks in advance to the maintainer or Hackage trustee that gets this done :heart:

**Comment by andreasabel** [comment]
ATTN: @expipiplus1

**Comment by andreasabel** [comment]
I published this relaxation as revision 1 of 0.9.2.

**Comment by frasertweedale** [comment]
Thank you!
